### DISCUSSION TITLE
Does pgx support multiple host:port in database connection string?

### DISCUSSION THREAD

**Discussion by antonywhatever** [jackc/pgx#1608] [Q&A]
Does pgx support PostgreSQL connection string like this?
`postgres://node1.FQDN:5432,node2.FQDN:5432,node3.FQDN:5432/db_name?target_session_attrs=read-write`

For more context, we use pgx as part of Ory Hydra.

**Comment by jackc** [comment]
Yes. See https://pkg.go.dev/github.com/jackc/pgx/v5@v5.3.1/pgconn#ParseConfig.
